autogen/safer-cluster/main.tf.tmpl

```diff
@@ -130,9 +130,7 @@ module "gke" {
 
   // Define PodSecurityPolicies for differnet applications.
   // Example: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#example
-  pod_security_policy_config = [{
-    "enabled" = true
-  }]
+  pod_security_policy_config = var.pod_security_policy_config
```

Comment:
This is a selfish change, but at least the default remains the same. The devs using the cluster I've configured from this aren't ready to deal with a pod security policy just yet, so I can understand if you don't want this in the final pull request.